spec/index.html

```diff
@@ -1084,8 +1084,8 @@ <h3>The XML Schema Built-in Datatypes</h3>
       <tr><td><a data-cite="xmlschema11-2#integer"><code>xsd:integer</code></a></td><td>Arbitrary-size integer numbers</td></tr>
 
       <tr><th rowspan="2">IEEE floating-point<br />numbers</th>
-          <td><a data-cite="xmlschema11-2#double"><code>xsd:double</code></a></td><td>64-bit floating point numbers incl. ±Inf, ±0, NaN</td></tr>
-      <tr><td><a data-cite="xmlschema11-2#float"><code>xsd:float</code></a></td><td>32-bit floating point numbers incl. ±Inf, ±0, NaN</td></tr>
+          <td><a data-cite="xmlschema11-2#double"><code>xsd:double</code></a></td><td>64-bit floating point numbers incl. ±Inf, ±0, NaN (note that this does <em>not</em> include all decimal numbers)</td></tr>
```

Comment:
I don't think text here is necessary because it already says "64-bit floating point numbers". Having the note below is enough.

Reply:
> I don't think text here is necessary because it already says "64-bit floating point numbers". Having the note below is enough.

Okay, I will remove these parentheses again.